mlflow/models/docker_utils.py

```diff
@@ -31,7 +31,9 @@
     https://github.com/pyenv/pyenv.git /root/.pyenv
 ENV PYENV_ROOT="/root/.pyenv"
 ENV PATH="$PYENV_ROOT/bin:$PATH"
-RUN apt install -y python3.7
+RUN apt install -y software-properties-common \
+    && add-apt-repository -y ppa:deadsnakes/ppa \
+    && apt install -y python3.7
```

Comment:
I think in 20.04 the distutils package is no longer considered "core python". 
`apt install -y python3.7-distutils` (and maybe `apt install -y python3.7-dev` are required now?)

Reply:
I'll try `apt install -y python3.7-distutils` :)

Reply:
Added :)